dispatcher/frontend-ui/package.json

```diff
@@ -16,7 +16,7 @@
     "bootstrap-vue": "^2.1.0",
     "core-js": "^3.3.2",
     "deep-diff": "^1.0.2",
-    "filesize": "^6.0.1",
+    "filesize": "^6.4.0",
```

Comment:
Why not upgraded to latest version ? There are major changes but the API we are using seems pretty straightforward